src/components/backend-ai-session-list.ts

```diff
@@ -1100,14 +1100,15 @@ export default class BackendAiSessionList extends BackendAIPage {
                                @click="${(e) => this._openTerminateSessionDialog(e)}"
                                icon="delete"><wl-icon>delete</wl-icon></wl-button>
           ` : html``}
-          ${this._isRunning ? html`
+          ${this._isRunning || globalThis.backendaiclient.APIMajorVersion > 4 ? html`
```

Comment:
globalThis does not fire the changed event after component initialization so we need to create local variable to match with globalThis variable. 

e.g. 
```
firstUpdate() {
...
   document.addEventListener('backend-ai-connected', () => {
      this.switch1 = globalThis.switch1;
...
}
...
${this._isRunning || this.switch1 > 4 ? html`
```

Reply:
Would there be any case that APIMajorVersion can be changed after component initialization?

Reply:
I registered API major version as a property.